### ISSUE TITLE
Error when starting vim with .tex file

### ISSUE THREAD

**Issue by olmokramer** [machakann/vim-sandwich#49]
I get this error when I start vim (actually neovim) with a .tex file:

```
$ nvim test.tex
"test.tex" 1L, 4C
Error detected while processing function sandwich#util#addlocal:
line    2:
E121: Undefined variable: g:sandwich#recipes
E116: Invalid arguments for function deepcopy(g:sandwich#recipes)
E15: Invalid expression: deepcopy(g:sandwich#recipes)
```
I get this even with no configuration at all, except a clone of this repository at `$HOME/.config/nvim/pack/plugins/start/vim-sandwich`.

**Comment by albertfgu** [comment]
I had a similar problem at one point, which was a result of not finishing loading my plugin manager before the line with `deepcopy(g:sandwich#recipes)`. I don't know the order these things are called.

**Comment by machakann** [comment]
Thank you! Probably, it has fixed now on c71f96792209764a373530c2f6fcb8b46aeab6cc. Please update and test it.

**Comment by olmokramer** [comment]
Yep seems to work now. Thanks!
